packages/engine/src/getBlockMatcher.js

```diff
@@ -0,0 +1,60 @@
+/*
+  Copyright 2020-2021 Lowdefy, Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import { type } from '@lowdefy/helpers';
+
+const getBlockMatcher = (params) => {
+  let testParams = params;
+  if (type.isNone(testParams)) {
+    testParams = { blockIds: true };
```

Comment:
We can do this here:
```js
  if (type.isNone(testParams)) return () => true;
```
Skips a couple of extra if statements, and we can drop the `testParams.blockIds === true` check below if we want to